File: backend/src/Liturgy.Domain/Phase.cs
namespace Liturgy.Domain;

public class Phase
{
    public Guid Id { get; set; }
    public Guid ProjectId { get; set; }
    public PhaseKind Kind { get; set; }
    public PhaseState State { get; set; } = PhaseState.Locked;
    public int Order { get; set; }
    public DateTimeOffset CreatedAt { get; set; }
}

File: frontend/projects/liturgy/api/src/lib/models/phase.ts
import { Gate } from './gate';
import { PhaseKind } from './phase-kind';
import { PhaseState } from './phase-state';

export interface Phase {
  id: string;
  kind: PhaseKind;
  state: PhaseState;
  order: number;
  gate: Gate | null;
}
